File: ABC/python/059/B.py
import sys
A = int(input())
B = int(input())

if A < 0 or A > 10 ** 100 or B < 0 or B > 10 ** 100:
    sys.exit()

if A - B > 0:
    print("GREATER")
if A - B == 0:
    print("EQUAL")
if A - B < 0:
    print("LESS")

File: ABC/ruby/059/B.rb
A = gets().chomp.to_i
B = gets().chomp.to_i

exit if !( 1 <= A && A <= 10**100 && 1 <= B && B <= 10**100 )
puts  A == B ? 'EQUAL' : A > B ? 'GREATER' : 'LESS'